php-classes/Slate/CBL/Tasks/StudentTasksRequestHandler.php

```diff
@@ -501,6 +528,19 @@ public static function getRequestedArchiveFilter($fieldName = 'include_archived'
         return !!$_REQUEST[$fieldName];
     }
 
+    public static function getRequestedStudentDisabledFilter($fieldName = 'include_deactivated_students')
```

Comment:
```suggestion
    public static function getRequestedIncludeDeactivated($fieldName = 'include_deactivated')
```

let's rename this param to `include_deactivated` so we can be consistent with the name of this filter across endpoints which are not all scoped to students. Deactivation applies to profiles of all types of which student is only one

Also, move this method up to `\Slate\RecordsRequestHandler`